shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/base/BaseITCase.java

```diff
@@ -188,43 +190,57 @@ private String getActualJdbcUrlTemplate(final String databaseName) {
         }
     }
     
-    protected void initShardingAlgorithm() throws InterruptedException {
-        executeWithLog(getCommonSQLCommand().getCreateDatabaseShardingAlgorithm());
-        TimeUnit.SECONDS.sleep(2);
-        executeWithLog(getCommonSQLCommand().getCreateOrderShardingAlgorithm());
-        TimeUnit.SECONDS.sleep(2);
-        executeWithLog(getCommonSQLCommand().getCreateOrderItemShardingAlgorithm());
+    protected void initShardingAlgorithm() {
+        executeWithLogAndWait(getCommonSQLCommand().getCreateDatabaseShardingAlgorithm());
+        executeWithLogAndWait(getCommonSQLCommand().getCreateOrderShardingAlgorithm());
+        executeWithLogAndWait(getCommonSQLCommand().getCreateOrderItemShardingAlgorithm());
     }
     
     protected void getCreateOrderWithItemSharingTableRule() {
-        executeWithLog(commonSQLCommand.getCreateOrderWithItemSharingTableRule());
+        executeWithLogAndWait(commonSQLCommand.getCreateOrderWithItemSharingTableRule());
         assertBeforeApplyScalingMetadataCorrectly();
     }
     
     protected void createOrderSharingTableRule() {
-        executeWithLog(commonSQLCommand.getCreateOrderShardingTableRule());
+        executeWithLogAndWait(commonSQLCommand.getCreateOrderShardingTableRule());
     }
     
     protected void bindingShardingRule() {
-        executeWithLog("CREATE SHARDING BINDING TABLE RULES (t_order,t_order_item)");
+        executeWithLogAndWait("CREATE SHARDING BINDING TABLE RULES (t_order,t_order_item)");
     }
     
     protected void createScalingRule() {
-        executeWithLog("CREATE SHARDING SCALING RULE scaling_manual (INPUT(SHARDING_SIZE=1000), DATA_CONSISTENCY_CHECKER(TYPE(NAME=DATA_MATCH)))");
+        executeWithLogAndWait("CREATE SHARDING SCALING RULE scaling_manual (INPUT(SHARDING_SIZE=1000), DATA_CONSISTENCY_CHECKER(TYPE(NAME=DATA_MATCH)))");
     }
     
     protected void createSchema(final String schemaName) {
-        executeWithLog(String.format("CREATE SCHEMA %s", schemaName));
+        executeWithLogAndWait(String.format("CREATE SCHEMA %s", schemaName));
+    }
+    
+    protected void executeWithLogAndWait(final Connection connection, final String sql) throws SQLException {
+        log.info("connection execute:{}", sql);
+        connection.createStatement().execute(sql);
+        ThreadUtil.sleep(TimeUnit.SECONDS, 1);
     }
     
-    protected void executeWithLog(final String sql) {
+    protected void executeWithLogAndWait(final String sql) {
         log.info("jdbcTemplate execute:{}", sql);
         jdbcTemplate.execute(sql);
+        ThreadUtil.sleep(TimeUnit.SECONDS, 2);
     }
```

Comment:
`executeWithLogAndWait` could be simplified, since wait might be needed temporarily.